utils/base_tests.py

```diff
@@ -162,3 +163,68 @@ def key_should_not_accept_incorrect_input(self, field, value, error_message=None
                     ValueError,
             ):
                 model = self.initiate_model(field, value)
+
+
+class BaseSerializerTestCase(TestCase):
+
+    serializer_class = None
+    required_input = {}             # dict containing fields and values for successful validation
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        assert issubclass(self.serializer_class, BaseSerializer)
+        assert isinstance(self.required_input, dict)
+
+    """
+    Create serializer with one field altered from required_input.
+    """
+    def initiate_serializer(self, field, value):
+        values = self.required_input.copy()
+        values[field] = value
+        return self.serializer_class(data=values)
+
+    """
+    Create serializer from required_input.
+    """
+    def default_serializer(self):
+        return self.serializer_class(data=self.required_input)
+
+    """
+    Private method containing base code for running serializer validation tests.
+    """
+    def __field_input_acceptance_test(self, field, value, is_valid):
```

Comment:
One underscore is enough:
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables